src/integrationTest/java/eu/xenit/gradle/testrunner/ExampleRunner.java

```diff
@@ -98,4 +101,16 @@ public void publishWar() throws IOException {
         testProjectFolder(EXAMPLES.resolve("publish-war"), ":publishToMavenLocal");
     }
 
+    @Test
+    public void log4jPrefix() throws IOException {
+        testProjectFolder(EXAMPLES.resolve("log4j-prefix"), ":functionalTest");
+    }
+
+    @Test
+    public void log4jPrefixDisable() throws IOException {
+        assumeTrue("Gradle 6.2 is required to disable prefixLog4j tasks",
```

Comment:
Interesting. Does this test run with multiple gradle versions?

Reply:
Yes, these testcases get run against all supported Gradle versions.

This is handled in the parent class of these tests, with a parameterized runner: https://github.com/xenit-eu/alfresco-docker-gradle-plugin/blob/89365d6a860c296fa9e210f9ea25393e7125b65d/src/integrationTest/java/eu/xenit/gradle/testrunner/AbstractIntegrationTest.java#L46-L58

We can only compile against one Gradle version, so it is important to run integration tests against all versions that we want to support, to ensure that we don't use functionality that breaks in other (major) versions.